build.sbt

```diff
@@ -1,6 +1,6 @@
-ThisBuild / scalaVersion       := "2.13.1"
```

Comment:
I think you forgot to call `fix`...

Reply:
![yes](https://user-images.githubusercontent.com/3316502/75708659-88b18400-5cc1-11ea-80f6-ba513c39c06a.gif)